ere-guests/Cargo.toml

```diff
@@ -191,6 +192,14 @@ k256 = { version = "0.13", default-features = false, features = [
 ] }
 sparsestate = { git = "https://github.com/eth-act/zkvm-ethereum-mpt.git", tag = "v0.3.0" }
 
+[profile.release]
+lto = true
+panic = "abort"
+codegen-units = 1
+
+[profile.release.package.reth-airbender-stateless-validator]
+opt-level = "s"
```

Comment:
Is this a recommendation from airbender?

Reply:
Ah no, it's because `opt-level = 3` still generate binary > 2MiB, so I have to use either `s` or `z` to optmize for binary size specifically, and `s` is small enough so I picked `s`.